test/spec/startup_script_stdlib_version_spec.bats

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
```

Comment:
Since we don't have defined boilerplate for `.bats` files I think `verify_boilerplate.py` missed this boilerplate. Should this be the standard Google Apache 2 license header?

Reply:
Yes, good catch.